WifiControl.cpp

```diff
@@ -106,3 +119,21 @@ String WifiControl::getSoftwareMacAddress(const String& delimiter)
 	WiFi.macAddress(rawMac);
 	return format6Bytes(rawMac, delimiter);
 }
+
+String WifiControl::generateRandomSecret(unsigned length) const
+{
+	// There is no "O" (Oh) to reduce confusion
+	const String validChars{"abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNPQRSTUVWXYZ0123456789.-,:!$/"};
```

Comment:
i I, L l, ! 1 and 0 should also be removed to reduce confusion to the absolute minimum ;)

Reply:
Agree. Will update the PR accordingly this evening.

Reply:
Done. https://github.com/Dirk007/Basecamp/commit/df167fe832728a941ec0c8323a1da2bff3867422